FORMAT_ADIOS.md

```diff
@@ -32,3 +32,35 @@ Output from `bpls -A` for a boolean attribute `pybool` stored in the location of
 
 There is no convention yet for a unique representation of ADIOS2 variables with boolean type.
 Thus, implementations should cast the data to and from `unsigned char` instead.
+
+## `stepBased` Encoding of Iterations
+
+In order to correlate openPMD iterations with ADIOS steps, the *root* group (path `/`) in ADIOS must contain a variable:
+
+  - `__step__`
+    - type: 1-dimensional array containing N *(int)* elements, where N is the number of ADIOS steps
+    - description: for each ADIOS step, this variable needs to be updated with the corresponding openPMD iteration.
+    - note: ADIOS steps are absolute and not every ADIOS step or openPMD iteration contains an update for each declared openPMD record.
+    - advice to implementers: [decide on this] an openPMD iteration for different openPMD records might be spread over multiple ADIOS steps.
```

Comment:
@franzpoeschel let us discuss if we like to provide the freedom expressed in the first sentence tomorrow

Reply:
Offline discussed: let's keep this open for now but we don't implement it yet (could be tricky in streaming, we might need more boundary conditions)